src/constants/endpoints.ts

```diff
@@ -0,0 +1,22 @@
+import config from '../config'
+
+export default {
+  LOGIN: config.API_BASE_URL + '/api/v1/token/',
```

Comment:
We shouldn't use absolute URLs (extended with `API_BASE_URL`)
Axios is already preconfigured to call `API_BASE_URL` if the relative URL address is passed.